File: lexilogicserver/src/main/java/org/amalgam/lexilogicserver/model/utilities/referenceobjects/GameDetail.java
package org.amalgam.lexilogicserver.model.utilities.referenceobjects;

public class GameDetail {
    private String username;
    private int lobbyID;
    private int totalPoints;

    public GameDetail() {
    }

    public GameDetail(String username, int lobbyID, int totalPoints) {
        this.username = username;
        this.lobbyID = lobbyID;
        this.totalPoints = totalPoints;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public int getLobbyID() {
        return lobbyID;
    }

    public void setLobbyID(int lobbyID) {
        this.lobbyID = lobbyID;
    }

    public int getTotalPoints() {
        return totalPoints;
    }

    public void setTotalPoints(int totalPoints) {
        this.totalPoints = totalPoints;
    }
}

File: Client_Python/Utilities/Reference_Objects/GameDetail.py
class GameDetail:
    def __init__(self, username=None, lobby_id=None, total_points=None):
        self._username = username
        self._lobby_id = lobby_id
        self._total_points = total_points

    def get_username(self):
        return self._username

    def set_username(self, value):
        self._username = value

    def get_lobby_id(self):
        return self._lobby_id

    def set_lobby_id(self, value):
        self._lobby_id = value

    def get_total_points(self):
        return self._total_points

    def set_total_points(self, value):
        self._total_points = value